src/main/java/de/bonndan/nivio/input/kubernetes/KubernetesObserver.java

```diff
@@ -0,0 +1,77 @@
+package de.bonndan.nivio.input.kubernetes;
+
+
+import de.bonndan.nivio.model.Landscape;
+import de.bonndan.nivio.observation.InputChangedEvent;
+import de.bonndan.nivio.observation.InputFormatObserver;
+import de.bonndan.nivio.observation.ObservedChange;
+import io.fabric8.kubernetes.client.KubernetesClient;
+import io.fabric8.kubernetes.client.KubernetesClientException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.context.ApplicationEventPublisher;
+import org.springframework.lang.NonNull;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+
+public class KubernetesObserver implements InputFormatObserver {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(KubernetesObserver.class);
+
+    private final Landscape landscape;
+    private final ApplicationEventPublisher eventPublisher;
+    private final KubernetesClient kubernetesClient;
+    private final List<Long> eventUidList;
+
+    public KubernetesObserver(@NonNull final Landscape landscape,
+                              @NonNull final ApplicationEventPublisher eventPublisher,
+                              @NonNull final KubernetesClient kubernetesClient) {
+        this.landscape = landscape;
+        this.kubernetesClient = kubernetesClient;
+        this.eventPublisher = eventPublisher;
+        this.eventUidList = getK8sComponents();
+    }
+
+    @Override
+    public void run() {
+        var change = false;
+        while (!change) {
+            sleep(1000);
```

Comment:
The observers are started in a thread pool with scheduleWithFixedDelay. This basically means that we do not have to sleep here. However, the given delay (30s I think) might be a bit long. Feel free to change the delay (maybe on a per-observer basis).